src/lang/std/sketchcombos.ts

```diff
@@ -1559,7 +1559,9 @@ export function transformSecondarySketchLinesTagFirst({
     }
   | Error {
   // let node = structuredClone(ast)
-  const primarySelection = selectionRanges.codeBasedSelections[0].range
+  const primarySelection = selectionRanges.codeBasedSelections.sort(
```

Comment:
[`sort()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) mutates the original array. Maybe we shouldn't do that.